### PR TITLE
deps: fix the rgb-lib version

### PR DESCRIPTION
**Pull request by vincenzopalazzo**
```
➜  rgb-lightning-node git:(master) cargo install --debug --path .
  Installing rgb-lightning-node v0.1.0 (/home/vincent/github/work/rgb-tools/rgb-lightning-node)
    Updating crates.io index
    Updating git repository `https://github.com/RGB-Tools/rgb-lib`
    Updating git repository `https://github.com/arik-so/rust-musig2`
error: failed to compile `rgb-lightning-node v0.1.0 (/home/vincent/github/work/rgb-tools/rgb-lightning-node)`, intermediate artifacts can be found at `/home/vincent/github/work/rgb-tools/rgb-lightning-node/target`

Caused by:
  failed to select a version for `strict_encoding`.
      ... required by package `lightning v0.0.118 (/home/vincent/github/work/rgb-tools/rgb-lightning-node/rust-lightning/lightning)`
      ... which satisfies path dependency `lightning` of package `rgb-lightning-node v0.1.0 (/home/vincent/github/work/rgb-tools/rgb-lightning-node)`
  versions that meet the requirements `=2.6.1` are: 2.6.1

  all possible versions conflict with previously selected packages.

    previously selected package `strict_encoding v2.6.2`
      ... which satisfies dependency `strict_encoding = "=2.6.2"` of package `rgb-lightning-node v0.1.0 (/home/vincent/github/work/rgb-tools/rgb-lightning-node)`
```

### CODE REVIEW

File: lightning/Cargo.toml [hunk 1]
```diff
@@ -59,12 +59,12 @@ hex = "0.4"
...
 rgb-contracts = { version = "=0.10.2", features = ["electrum"] }
 rgb_core = { package = "rgb-core", version = "=0.10.8" }
-rgb-lib = { git = "https://github.com/RGB-Tools/rgb-lib", branch = "master" }
+rgb-lib = { git = "https://github.com/RGB-Tools/rgb-lib", branch = "rln_v0.10" }
 rgb-std = "=0.10.9"
 rgb-wallet = "=0.10.9"
 serde = { version = "^1.0", features = ["derive"] }
 serde_json = "^1.0"
-strict_encoding = "=2.6.1"
+strict_encoding = "=2.6.2"
```
Comment (zoedberg) [inline lightning/Cargo.toml]: please revert this change, I think it's better to use the same version that rgb-lib uses in the branch `rln_v0.10`

**Review by zoedberg — CHANGES_REQUESTED** [review]

File: lightning/Cargo.toml (hunk 1 above)
Comment (vincenzopalazzo) [reply lightning/Cargo.toml]: Sorry I forget to review this! Done in https://github.com/RGB-Tools/rust-lightning/pull/3/commits/b479dfceab80403a32296198ecd2d0bb4e4ba885

**Review by zoedberg — APPROVED** [review]
ACK b479dfceab80403a32296198ecd2d0bb4e4ba885

Thanks!
